include/cantera/thermo/MargulesVPSSTP.h

```diff
@@ -341,12 +338,12 @@ class MargulesVPSSTP : public GibbsExcessVPSSTP
 
     virtual void getdlnActCoeffdT(doublereal* dlnActCoeffdT) const;
 
-    /// @}
-    /// @name Initialization The following methods are used in the process of
-    ///     constructing the phase and setting its parameters from a
-    ///     specification in an input file. They are not normally used in
-    ///     application programs. To see how they are used, see importPhase()
-    /// @{
+    //! @}
+    //! @name Initialization The following methods are used in the process of
+    //!     constructing the phase and setting its parameters from a
+    //!     specification in an input file. They are not normally used in
+    //!     application programs. To see how they are used, see importPhase()
+    //! @{
```

Comment:
While you're at it, you could fix the formatting of this heading -- presumably, only `Initialization` is meant to be the heading, rather than `Initialization The following methods are used in the process of`, which is what gets put in the HTML docs.